test/integration/ads_integration_test.cc

```diff
@@ -55,6 +56,42 @@ TEST_P(AdsIntegrationTest, BasicClusterInitialWarming) {
   test_server_->waitForGaugeGe("cluster_manager.active_clusters", 2);
 }
 
+TEST_P(AdsIntegrationTest, UpdateToSubsetOfResources) {
```

Comment:
Please add a comment describing the test.

Reply:
done